server/task/task-memory/src/main/java/org/apache/james/task/SerialTaskManagerWorker.java

```diff
@@ -105,21 +106,21 @@ private void handleExecutionError(TaskWithId taskWithId, Listener listener, Thro
     }
 
     private Task.Result run(TaskWithId taskWithId, Listener listener) {
-        listener.started(taskWithId.getId());
+        Mono.from(listener.started(taskWithId.getId())).block();
```

Comment:
you can flatmap or use then on the result of this Mono ?
Maybe you can remove some other block below too

Reply:
I thought of it, but it ends up in a `CompletableFuture` and it looks complex to change that.